code/src/lib/dbConnect.ts

```diff
@@ -0,0 +1,60 @@
+import mongoose from 'mongoose';
```

Comment:
The caching thing here is weird

Reply:
I did this because we were establishing a new connection to the DB each time we got a request. I followed the "official" [guide](https://github.com/vercel/next.js/blob/canary/examples/with-mongodb-mongoose/lib/dbConnect.ts), but tbh don't understand every line fully.